asmcomp/power/emit.mlp

```diff
@@ -521,6 +536,35 @@ let emit_poll env i return_label far =
     | Some return_label -> ` b   {emit_label return_label}\n`
   end
 
+let bound_error_label env dbg =
+  if !Clflags.debug then begin
```

Comment:
I am not familiar with this code. I'm reviewing the code by comparing this with the arm64 backend. I wondered why this code is different from the arm64 version of this function:

https://github.com/ocaml/ocaml/blob/22084b7ac6984ef5f5392a56849365dab16a4151/asmcomp/arm64/emit.mlp#L164-L175

Reply:
To be frank, the ARM64 code smells, e.g. the use of `List.hd`.  The code I wrote for POWER is clearer, if I can say so myself.

Reply:
For the record, I intend to work on unifying this logic between platforms (but not in this PR). I have also noticed riscv64 will always create new labels regardless of `Cflags.debug`...

Reply:
Thanks @dustanddreams. This would be useful.